docker-compose.yml

```diff
@@ -114,9 +114,6 @@ services:
   traefik:
     image: traefik:1.7
     restart: always
-    ports:
-      - 80:80
-      - 443:443
```

Comment:
Correct me if I'm wrong, but as far as I'm aware this is exposing local ports to the host machine, but in our case only other containers are accessing it, so there's no reason to expose these ports to the host.

Besides, this was removed because docker listens on both ports, and this mapping was creating a conflict.

Reply:
> Correct me if I'm wrong, but as far as I'm aware this is exposing local ports to the host machine, but in our case only other containers are accessing it, so there's no reason to expose these ports to the host.

No that is only true for the other containers. Traefik is the reverse proxy, which makes it what gets accessed from the outside.

> Besides, this was removed because docker listens on both ports, and this mapping was creating a conflict.

Docker doesn't listen to those ports normally.
I assume this was on your local machine? Then it's probably some other (possibly old / forgotten) containers running on those ports.

Reply:
There's definitely something weird going on on your system:

This is also on macOS:
```
ulo@carbon ~
○ sudo lsof -PniTCP -sTCP:LISTEN | grep -e ':443'

ulo@carbon ~
○ docker run --rm -d --name test -p 443:443 busybox sleep 100
35cf60a39339c58a2e86e993bf100406b612f274501a2a64efd7f7acca58c774

ulo@carbon ~
○ sudo lsof -PniTCP -sTCP:LISTEN | grep -e ':443'
com.docke 81713  ulo  101u  IPv6 0xacb4590c5987f92b      0t0  TCP *:443 (LISTEN)
```